sources/packages/web/src/components/aest/students/assessment/Request.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <v-container>
+    <v-card>
+      <v-container>
+        <body-header title="Requests" class="m-1"> </body-header>
```

Comment:
Can we please add the _recordsCount_ also here?

Reply:
added